php/context/class-fieldmanager-context-post.php

```diff
@@ -310,6 +310,12 @@ protected function add_data( $post_id, $meta_key, $meta_value, $unique = false )
 	 *                                Default empty.
 	 */
 	protected function update_data( $post_id, $meta_key, $meta_value, $data_prev_value = '' ) {
+		// Meta is always stored as a string, so if this is a scalar value, cast
+		// it as a string to ensure that `update_metadata()` is able to correctly
+		// compare the current value against the previous value.
```

Comment:
Perhaps should follow docs standards for inline comments: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-2-multi-line-comments (applies throughout)?